### PR TITLE
[FIX] website_sale_stock: qweb view_product_availability.xml 404 (NOT FOUND)

### PR DESCRIPTION
**Pull request by hugho-ad**
image

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

OPW
1966003

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_sale_stock/__manifest__.py [hunk 1]
```diff
@@ -16,5 +16,8 @@
         'views/product_template_views.xml',
         'views/res_config_settings_views.xml',
         'views/website_sale_stock_templates.xml'
-    ]
+    ],
+    'qweb': [
+        'static/src/xml//website_sale_stock_product_availability.xml',
```
Comment (mart-e) [inline addons/website_sale_stock/__manifest__.py]: `xml//website` ?

**Comment by mart-e** [comment]
Actually it should be loaded correctly thanks to this line

https://github.com/odoo/odoo/blob/610ab82b51937beccdc3e69dc55b6f843047cae5/addons/website_sale_stock/static/src/js/website_sale_stock.js#L10

How do you reproduce this bug? I made a quick test on a [runbot](http://runbot.odoo.com/runbot/1/11.0) and the file was loaded correctly

File: addons/website_sale_stock/__manifest__.py (hunk 1 above)
Comment (hugho-ad) [reply addons/website_sale_stock/__manifest__.py]: Ups, thanks!

**Comment by hugho-ad** [comment]
You rigth thanks!

**Pull request closed without merging**